src/main/scala/com/advancedtelematic/campaigner/http/UpdateResource.scala

```diff
@@ -53,8 +51,6 @@ class UpdateResource(extractNamespace: Directive1[Namespace], deviceRegistry: De
     HypermediaResource(links, update)
   }
 
-  val groupUpdateResolver = new GroupUpdateResolver(deviceRegistry, resolver)
```

Comment:
Why not  using the variable?

Reply:
Right, that got lost somewhen while refactoring. I brought it back now :)